pages/collections/_type/_.vue

```diff
@@ -241,6 +241,14 @@
           && this.page.relatedLinksCollection.items.length > 0)
           ? this.page.relatedLinksCollection.items : null;
       },
+      relatedCollectionsFound() {
+        if (this.relatedEntities !== null && this.relatedEntities.length > 0) {
+          return true;
+        } else if (this.relatedCollectionCards !== null && this.relatedCollectionCards.length > 0) {
+          return true;
+        }
+        return false;
+      },
```

Comment:
Perhaps you can use the optional chaining operator? Not sure, if we use that elsewhere already. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

`!this.relatedEntities?.length`

Reply:
thanks I have implemented this